### DISCUSSION TITLE
上色后导线无法与基础导线相连

### DISCUSSION THREAD

**Discussion by zgybd** [IndustrialCraftMC/IndustrialCraft2#132] [【漏洞反馈】Bugs Report]
1.19.2：industrialcraft-2-2.9.32+ex119-1.19.2-fabric
1.12.2：industrialcraft-2-2.8.222-ex112

导线上色后无法与基础导线相连
不清楚是不是新特性，但还是觉得与基础导线连上更好一点
**1.19.2**
![2691e70690b8a11a010860fa65ddb545](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/69385074/5af4ea18-5eb2-4c12-bfce-599c37638dd9)
**1.12.2**
![85aceb94284126326057b4a80c5bc700](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/69385074/61c5f536-27e6-46e0-b4ee-9b3271c64830)

**Comment by AbbottMc** [comment]
已修复
